ballroom-beats/src/components/DanceType/DanceType.js

```diff
@@ -1,22 +1,47 @@
 import React, { Component } from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import { View, Text, StyleSheet, Picker } from 'react-native';
 import { createStackNavigator } from 'react-navigation-stack';
 import { withNavigation } from 'react-navigation';
 import Icon from 'react-native-vector-icons/AntDesign';
+import Navbar from '../Navbar/Navbar';
 
 class DanceType extends Component {
+  constructor() {
+    super()
+    this.state= {
+        PickerSelectedDance : ''
+    }
+  }
+    // method for taking value, passing down to songPick for what songs to display
+    getSelectedPickerDance = () =>{
+    
+    }
 
   render() {
       return (
           <View style={styles.container}>
             <Text style={styles.dance}>Pick Your Dance Type</Text>
+            <View style={styles.pickerContainer}>
+                <Picker
+                    style={styles.picker}
+                    itemStyle={styles.picker_text}
+                    // style={{height: 30, width: 110}}
+                    selectedValue={this.state.PickerSelectedDance}
+                    onValueChange={(itemValue, itemIndex) => 
+                        this.setState({PickerSelectedDance: itemValue})} >
+                    <Picker.Item label="Beginner" value="Beginner" />
+                    <Picker.Item label="Intermediate" value="Intermediate" />
+                    <Picker.Item label="Advanced" value="Advanced" />
+                </Picker>
+            </View>
```

Comment:
@spaceplesiosaur The other change is DanceType, SongPick now has a picker component. It will need additional styling though